test/integration/load_stats_integration_test.cc

```diff
@@ -326,8 +328,19 @@ class LoadStatsIntegrationTest : public Grpc::GrpcClientIntegrationParamTest,
     result = upstream_request_->waitForEndStream(*dispatcher_);
     RELEASE_ASSERT(result, result.message());
 
+    // Send three metrics, one of which is not in the config.
+    xds::data::orca::v3::OrcaLoadReport orca_load_report;
```

Comment:
I think that adding this should be optional (controlled by an argument to the function).
The idea behind this is to make sure the code paths work when this is not reported, to make sure Envoy can still cope in case the report is not populated.

Reply:
Done.